backend/app/environment_platform_service_configurations/schemas/databricks_schema.py

```diff
@@ -0,0 +1,5 @@
+from app.shared.schema import ORMModel
+
+
+class DatabricksConfig(ORMModel):
+    pass_value: str
```

Comment:
have you tried this with an empty config? Would be cleaner

Reply:
Yup, it works!